include/swift/ABI/MetadataValues.h

```diff
@@ -1245,19 +1245,23 @@ class TypeContextDescriptorFlags : public FlagSet<uint16_t> {
     /// descriptor.  A TypeReferenceKind.
     ///
     /// Only meaningful for class descriptors.
-    Class_SuperclassReferenceKind = 10,
+    Class_SuperclassReferenceKind = 9,
     Class_SuperclassReferenceKind_width = 3,
 
     /// Whether the immediate class members in this metadata are allocated
     /// at negative offsets.  For now, we don't use this.
-    Class_AreImmediateMembersNegative = 13,
+    Class_AreImmediateMembersNegative = 12,
 
     /// Set if the context descriptor is for a class with resilient ancestry.
     ///
     /// Only meaningful for class descriptors.
-    Class_HasResilientSuperclass = 14,
+    Class_HasResilientSuperclass = 13,
 
-    /// Set if the context descriptor is includes metadata for dynamically
+    /// Set if the context descriptor includes metadata for dynamically
+    /// installing method overrides at metadata instantiation time.
+    Class_HasOverrideTable = 14,
```

Comment:
I mean, ok, but why not just add this at bit 9?

Reply:
You're right, I did more work here than necessary, but I'd like to not change it back and have to update the tests *again* :)

Reply:
Yeah, it's not really a problem, just curious if you had a motivation. :)